lib/conjur/command/ldapsync.rb

```diff
@@ -4,14 +4,16 @@ class Conjur::Command::LDAPSync < Conjur::Command
 
   LIST_FORMATS = %w(pretty json)
 
-  def self.find_job_by_id args
-    job_id = require_arg args, 'JOB-ID'
+  def self.error_messages(resp)
+    resp['events'].collect {|e| e['message'] if e['severity'] == 'error'}.compact
+  end
 
-    if (job = api.ldap_sync_jobs.find{|j| j.id == job_id})
-      job
-    else
-      exit_now! "No job found with ID '#{job_id}'"
-    end
+  def self.show_messages(resp)
+    msgs = resp['events'].collect do |e|
+      next unless e['severity'] == 'warn' || e['severity'] == 'error'
+      "\n" + e['severity'].upcase + ": " + e['message']
```

Comment:
I didn't think it was extra. It's the difference between
```
$ conjur ldap-sync policy show

WARN: something not so bad happened

WARN: another not so bad thing happend

ERROR: something really bad happened

error: Failed creating the policy.
```

and

```
$ conjur ldap-sync policy show
WARN: something not so bad happened
WARN: another not so bad thing happend
ERROR: something really bad happened
error: Failed creating the policy.
```

I thought the former was more readable, especially as the messages got longer.